packages/base/src/util/InvisibleMessage.js

```diff
@@ -0,0 +1,50 @@
+import InvisibleMessageMode from "../types/InvisibleMessageMode.js";
+import getSingletonElementInstance from "./getSingletonElementInstance.js";
+
+const styles = `position: absolute;
+	clip: rect(1px,1px,1px,1px);
+	user-select: none;
+	left: -1000px;
+	top: -1000px;
+	pointer-events: none;`;
+
+const politeSpan = document.createElement("span");
+const assertiveSpan = document.createElement("span");
+
+politeSpan.classList.add("ui5-invisiblemessage-polite");
+assertiveSpan.classList.add("ui5-invisiblemessage-assertive");
+
+politeSpan.setAttribute("aria-live", "polite");
+assertiveSpan.setAttribute("aria-live", "assertive");
+
+politeSpan.setAttribute("role", "alert");
+assertiveSpan.setAttribute("role", "alert");
+
+politeSpan.style.cssText = styles;
+assertiveSpan.style.cssText = styles;
+
+if (!politeSpan.parentElement) {
+	getSingletonElementInstance("ui5-static-area").appendChild(politeSpan);
+	getSingletonElementInstance("ui5-static-area").appendChild(assertiveSpan);
+}
+
+/**
+ * Inserts the string into the respective span, depending on the mode provided.
+ *
+ * @param {string} message String to be announced by the screen reader.
+ * @param {sap.ui.core.InvisibleMessageMode} mode The mode to be inserted in the aria-live attribute.
+ */
+const announce = (message, mode) => {
+	// If no type is presented, fallback to polite announcement.
+	const span = mode === InvisibleMessageMode.Assertive ? assertiveSpan : politeSpan;
+
+	// Set textContent to empty string in order to trigger screen reader's announcement.
+	span.textContent = "";
```

Comment:
Why you'd need to make it empty and then assign the message?

Reply:
If someone wants to announce the same message over and over again on some interaction, the screen reader won't be triggered if we don't empty the value.

Reply:
Are we sure that way, it'd be read out?